modules/prebidServerBidAdapter/index.js

```diff
@@ -830,6 +830,8 @@ const OPEN_RTB_PROTOCOL = {
           bidObject.creativeId = bid.crid;
           if (bid.burl) { bidObject.burl = bid.burl; }
           bidObject.currency = (response.cur) ? response.cur : DEFAULT_S2S_CURRENCY;
+          bidObject.meta = bidObject.meta || {};
+          if (bid.adomain) { bidObject.meta.advertiserDomains = bid.adomain; }
```

Comment:
Is `advertiserDomains` supposed to be an array?

I see the test has it `expect(response.meta.advertiserDomains[0]).to.equal('appnexus.com');` but just making sure that PBS is responding with an array on the `bid.adomain` key.

Reply:
it is where i see it and in the test spec

Reply:
see https://github.com/prebid/Prebid.js/blob/master/test/spec/modules/prebidServerBidAdapter_spec.js#L256